content/en/tracing/troubleshooting/_index.md

```diff
@@ -164,13 +164,22 @@ var tracer = Tracer.Create(isDebugEnabled: true);
 Tracer.Instance = tracer;
 ```
 
+The environment variable `DD_TRACE_DEBUG` can also be set to `true`.
+
 Location of the profiler log:
 
 | Platform | Path                                                          |
 |----------|---------------------------------------------------------------|
 | Linux    | `/var/log/datadog/dotnet-profiler.log`                        |
 | Windows  | `C:\ProgramData\Datadog .NET Tracer\logs\dotnet-profiler.log` |
 
+Location of additional logs when debug is enabled:
```

Comment:
I think we can just keep the new table below (and drop the old one above) and say that all logs (Profiler and Tracer) are saved there by default.

Reply:
+1 to Lucas's comment

Reply:
Thanks, I just removed the profiler log table. Can you check the latest wording?